src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -114,6 +114,18 @@ function PolicyAccountingPage({policy, connectionSyncProgress}: PolicyAccounting
     const connectedIntegration = accountingIntegrations.find((integration) => !!policy?.connections?.[integration]) ?? connectionSyncProgress?.connectionName;
     const policyID = policy?.id ?? '';
 
+    const policyConnectedToXero = connectedIntegration === CONST.POLICY.CONNECTIONS.NAME.XERO;
+
+    const tenants = useMemo<Tenant[]>(() => {
```

Comment:
`tenants` isn't getting populated for me. I have 2 organizations connected with Expensify on Xero.
But `tenants` is undefined.

<img width="805" alt="image" src="https://github.com/Expensify/App/assets/29673073/856b0f1a-9335-4bf5-a55f-b7e0af99bb6a">